mongodb/map.jinja

```diff
@@ -1,14 +1,19 @@
 {% set mongodb = salt['grains.filter_by']({
     'default': {
-        'pkg': 'mongodb',
-        'service': 'mongodb',
-        'conf_file': '/etc/mongodb/mongodb.conf',
-        'config': {
-            'key': 'value',
-        },
+      'key': '7F0CEB10',
+      'keyserver': 'keyserver.ubuntu.com',
+      'pkgs': [
+        {'mongodb-10gen': '*'},
+      ],
+      'cluster_key_file': '/etc/mongodb_key',
+      'bind_ip': '127.0.0.1',
+      'port': 12017,
+      'enable_journal': True,
```

Comment:
Should this be `'true'` instead of `True`? If the default rendition wants to be a string then it should be explicitly set as such here.